signac/common/config.py

```diff
@@ -143,9 +145,12 @@ def get_config(infile=None, configspec=None, *args, **kwargs):  # noqa: D103
 
 def load_config(root=None, local=False):
     """Load configuration, searching upward from a root path."""
+
     if root is None:
         root = os.getcwd()
+
     config = Config(configspec=cfg.split("\n"))
+
```

Comment:
Reverting whitespace changes.
```suggestion
    if root is None:
        root = os.getcwd()
    config = Config(configspec=cfg.split("\n"))
```